WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -248,17 +248,25 @@ def update_events
     end
   end
 
+  def get_nearby_competitions(competition)
+    nearby_competitions = competition.nearby_competitions(Competition::NEARBY_DAYS_WARNING, Competition::NEARBY_DISTANCE_KM_WARNING)[0, 10]
+    nearby_competitions.select!(&:isConfirmed?) unless current_user.can_view_hidden_competitions?
+    nearby_competitions || []
```

Comment:
I tried removing that line and got an error

>NoMethodError at /competitions/TestNearby2016/edit
>undefined method `empty?' for nil:NilClass

Further debugging revealed that something was wrong with the "unless ..." part:

>(byebug) nearby_competitions
  CACHE (0.0ms)  SELECT  `teams`.* FROM `teams` WHERE `teams`.`friendly_id` = 'software' LIMIT 1  [["friendly_id", "software"]]
  CACHE (0.0ms)  SELECT COUNT(*) FROM `team_members` WHERE `team_members`.`user_id` = 6144 AND (end_date IS NULL OR end_date > '2016-11-08') AND `team_members`.`team_id` = 6  [["user_id", 6144], ["team_id", 6]]
*** ActionController::ParameterMissing Exception: param is missing or the value is empty: competition

>nil

I couldn't figure out what controller or what action it means...